docs/developer_guide.md

```diff
@@ -230,21 +236,25 @@ sys	0m39.333s
 <details><summary>Run tests on an OpenVINO Model Server binary file</summary>
 
 1. To run tests on an OpenVINO Model Server binary file, use export to specify the following variable in `user_config.py` or in the environment. 
-Replace `"/home/example_path/ovms/bin/ovms"` with the path to your binary file:
+Replace `"/home/<example_path>/dist/<os_name>/ovms/bin/ovms"` with the path to your binary file:
 
+```bash 
+tar -xvzf dist/<os_name>/ovms.tar.gz -C dist/<os_name>/
 ```
-os.environ["OVMS_BINARY_PATH"] = "/home/example_path/ovms/bin/ovms"
+
+```bash
+echo 'os.environ["OVMS_BINARY_PATH"] = "'${PWD}'/dist/<os_name>/ovms/bin/ovms"' >> user_config.py
 ```
 
 2. The following command executed in the of OpenVINO Model Server binary file should return paths to the unpacked `lib` directory included in `ovms.tar.gz` (`ovms/bin/./../lib`).
-```
-ldd ./ovms
+```bash
+ldd dist/<os_name>/ovms/bin/ovms
 ```
 
 3. Otherwise use export to specify the following variable in `user_config.py` file or in the environment :
 
-```
-os.environ["LD_LIBRARY_PATH"] = "<path to ovms libraries>"
+```bash
+echo 'os.environ["LD_LIBRARY_PATH"] = "'${PWD}'/dist/<os_name>/ovms/lib"' >> user_config.py
```

Comment:
Will this work out of the box? Don't we need:
`import os` ?

Reply:
I removed echo command and went back to previous version, since user_config.py is not used anywhere in file. I also added export command for this variable.